tests/test_gail.py

```diff
@@ -112,8 +112,8 @@ def test_gail_callback(tmp_path):
     dataset = ExpertDataset(expert_path=EXPERT_PATH_PENDULUM, traj_limitation=10,
                             sequential_preprocessing=True, verbose=0)
     model = GAIL("MlpPolicy", "Pendulum-v0", dataset)
-    checkpoint_callback = CheckpointCallback(save_freq=500, save_path=str(tmp_path / 'logs/gail/'), name_prefix='gail')
-    model.learn(total_timesteps=1000, callback=checkpoint_callback)
+    checkpoint_callback = CheckpointCallback(save_freq=150, save_path=str(tmp_path / 'logs/gail/'), name_prefix='gail')
+    model.learn(total_timesteps=300, callback=checkpoint_callback)
```

Comment:
This was "broken" before, but could set `total_timesteps` to 400 so that we enforce checkpointcall to happen twice (to test if first time around it changes some variables which prevents from working on next call)